Makefile

```diff
@@ -121,3 +121,8 @@ watch_css:
 sync:
 	rsync -azru --delete --stats public/ ../prism-break-static/public/
 	(cd ../prism-break-static; git add -A; git commit -m 'regenerate'; git push)
+
+# Localizations
+
+localize_%:
+	lsc ./source/functions/find-missing-localizations.ls $*
```

Comment:
Also I would move this up to into the 'FUNCTIONS' section.

Sorry I haven't looked through this too much, but what output is  expected from this target? Should it be run every time language X is run? If so it should be added as a dependency to the `html_%` target, but we need to think about the order too. Does it have to run first or can it run any time?